### PR TITLE
Add file_size property to Episode, separate it from location. Fixes #…

### PR DESCRIPTION
**Pull request by medariox**
…5960

- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: medusa/tv/episode.py [hunk 1]
```diff
@@ -382,16 +388,35 @@ def location(self):
...
+        """Return the file size.
+
+        :return:
+        :rtype: int
+        """
+        return self._file_size
+
+    @file_size.setter
+    def file_size(self, size=None):
+        if size is not None:
```
Comment (medariox) [inline medusa/tv/episode.py]: Do we really want to allow the optional `size`? 🤔

**Comment by p0psicles** [comment]
@medariox Can we have this merged first #5795

File: medusa/tv/episode.py (hunk 1 above)
Comment (p0psicles) [reply medusa/tv/episode.py]: Yeah I'm also curious in which situations where we do have a file, but don't have a size?

**Comment by p0psicles** [comment]
I think we should do a merge develop to be sure

**Comment by p0psicles** [comment]
@medariox what's the status of this?

**Comment by medariox** [comment]
Closing this in favor of https://github.com/pymedusa/Medusa/pull/7102

**Pull request closed without merging**